Patch by Niko Tyni <ntyni@iki.fi> for perl-CGI-SpeedyCGI >= 2.22, which adds documentation
regarding the variables behaviour - as it is easily misunderstood. For further information,
see: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=204461

--- CGI-SpeedyCGI-2.22/src/SpeedyCGI.src		2003-10-12 05:37:30.000000000 +0200
+++ CGI-SpeedyCGI-2.22/src/SpeedyCGI.src.documentation	2008-05-04 20:05:33.000000000 +0200
@@ -250,6 +250,13 @@
 be Speedy followed by the option name.  For example to set the
 Timeout option, use the apache directive SpeedyTimeout.
 
+Note that these variables are global. There is currently no way to run
+different scripts with different SpeedyCGI options when they are run
+from the Apache module. Any <Directory> or <Location> contexts
+have no effect on the scope of the SpeedyCGI options. When the
+same SpeedyCGI option is set several times, the last one overrides
+the others.
+
 =back
 
 =head2 Context
